src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,77 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector INJECTOR = Injector.getInstance("mate.jdbc");
+    private static final ManufacturerService MANUFACTURER_SERVICE = (ManufacturerService)
+            INJECTOR.getInstance(ManufacturerService.class);
+    private static final DriverService DRIVER_SERVICE = (DriverService)
+            INJECTOR.getInstance(DriverService.class);
+    private static final CarService CAR_SERVICE = (CarService)
+            INJECTOR.getInstance(CarService.class);
```

Comment:
These classes are not deeply immutable you should name these constants as you would any other variable, meaning don't use UPPER_CASE, please correct me if I'm wrong